### PR TITLE
Web 46/navbar changes

### PR DESCRIPTION
**Pull request by burtonjong**
PR for web46 navbar changes

### CODE REVIEW

File: components/Navbar.jsx [hunk 1]
```diff
@@ -59,6 +32,15 @@ const Navbar = () => {
           {/* <Link href="/partnerships">
             <span className={TEXT_CLASSES}>Partnerships</span>
           </Link> */}
```
Comment (topanb11) [inline components/Navbar.jsx]: can you delete the commented out code from 24-34? I think it should be safe to delete

File: components/Navbar.jsx [hunk 2]
```diff
@@ -59,6 +32,15 @@ const Navbar = () => {
...
+          <Link href="/who-we-are">
+            <span className={TEXT_CLASSES}>About</span>
+          </Link>
+          <Link href="/events">
+            <span className={TEXT_CLASSES}>Events</span>
+          </Link>
+          <Link href="/projects">
+            <span className={TEXT_CLASSES}>Project</span>
+          </Link>
```
Comment (topanb11) [inline components/Navbar.jsx:32]: I think you'll also have to add these links in `MobileNavbarDrawer.jsx` and remove the extra dropdown (I did not make it like that, there just so happens to be two nabbers)

**Review by topanb11 — COMMENTED** [review]
Thanks Burton! Just left a few comments, looks good on desktop so far and all the nav links work as expected!
Screenshot 2024-01-08 at 8 39 37 PM

Just need to update it on the mobile view then it should be good
Screenshot 2024-01-08 at 8 40 03 PM

Screenshot 2024-01-08 at 8 40 32 PM

File: components/Navbar.jsx (hunk 2 above)
Comment (topanb11) [reply components/Navbar.jsx:32]: There's some dead code in that file too you could probably get rid of as well

**Comment by burtonjong** [comment]
For the mobile navbar I did not change the anchor tags to Links as the navbar needs to re-render for it to dissapear after clicking an option in the navbar ( the react "Link" does not rerender a component but an anchor tag does ) 
image

**Comment by burtonjong** [comment]
Other than that all the links should redirect to the correct endpoint and it looks good
image

File: package.json [hunk 3]
```diff
@@ -30,7 +30,7 @@
     "eslint": "8.20.0",
     "eslint-config-next": "12.2.3",
     "postcss": "^8.4.14",
-    "tailwindcss": "^3.1.6"
+    "tailwindcss": "^3.3.6"
```
Comment (topanb11) [inline package.json]: I just pushed up some changes on main to fix this so it doesn't have to be changed every time, could you try rebasing your branch?

File: components/Navbar.jsx [hunk 4]
```diff
@@ -40,25 +21,15 @@ const Navbar = () => {
...
-            titleClasses={TEXT_CLASSES}
-            optionsClasses={OPTIONS_CLASSES}
-            optionClasses={OPTION_CLASSES}
-          >
-            <NavbarDropdownOptions options={["Announcements", "Blog"]} />
-          </Dropdown> */}
-          {/* <Link href="/partnerships">
-            <span className={TEXT_CLASSES}>Partnerships</span>
-          </Link> */}
+          <Link href="/who-we-are">
```
Comment (topanb11) [inline components/Navbar.jsx]: When running the app using `npm run dev` and pressing `About` I get an error like so
Screenshot 2024-01-10 at 5 40 17 PM
But if I run `npm run start` it loads normally, do you have this issue as well?

File: components/MobileNavbarDrawer.jsx [hunk 5]
```diff
@@ -1,56 +1,20 @@
...
-  };
-
   return (
     <div className="flex flex-col bg-[#FF4D6F] rounded-b-2xl -mt-5 p-10 absolute top-full w-full items-center z-10">
       <div className="flex flex-col w-full space-y-4">
-        <h1
-          className={`cursor-pointer ${TEXT_CLASSES} ${selectedLink === ABOUT ? "underline" : ""}`}
-          onClick={() => handleClick(ABOUT)}
-        >
+        <a className={TEXT_CLASSES} href="who-we-are">
```
Comment (topanb11) [inline components/MobileNavbarDrawer.jsx:9]: same issue on mobile 🤔

**Review by topanb11 — COMMENTED** [review]
thanks Burton! Navbar looks good on mobile! Just left a small comment and two questions

**Comment by burtonjong** [comment]
I rebased the branch off main again to accomadate the package.json changes and ran npx @next/codemod new-link . to accomodate for the legacybehaviour of the <Link> components. Now when the about section is pressed, it renders correctly. 
image

**Review by topanb11 — APPROVED** [review]
Thanks Burton! Tested and navbar looks good on desktop/mobile and links work! LGTM 🔥
Screenshot 2024-01-16 at 11 19 41 PM
Screenshot 2024-01-16 at 11 19 53 PM

There is an error when you try to click `Join` but it's on the main branch so I'll write up a ticket to address that issue
Screenshot 2024-01-16 at 11 22 20 PM

**Pull request merged**